man/chpl.txt

```diff
@@ -561,7 +541,27 @@ OPTIONS
                                   $CHPL_WIDE_POINTERS environment variable
                                   (defaults to 'struct').
 
-ENVIRONMENT VARIABLES FOR OPTIONS
+  Compiler Information Options
+
+  --copyright     Print the compiler's copyright information.
+
+  -h, --help      Print a list of the command line options, indicating
+                  the arguments that they expect and a brief summary of their
+                  purpose.
+
+  --help-env      Print the command line option help message, listing
+                  the environment variable equivalent for each flag (see
+                  ENVIRONMENT VARIABLES FOR OPTIONS) and its current value.
+
+  --help-settings   Print the command line option help message, listing
+                 the current setting of each option as specified by
+                 environment variables and other flags on the command line.
+
+  --license       Print the compiler's license information.
+
+  --version       Print the version number of the compiler.
+
+ENVIRONMENT
 
   Most compiler command-line options have an environment variable that can
   be used to specify a default value. Use the --help-env option to list the
```

Comment:
Arguably not your problem (and maybe @mppf's), but the next few lines after this say "Command-line options will always override environment variable settings in the event of a conflict." but I think we established that this wasn't the case for --ccflags and --ldflags (?).